### PR TITLE
Linter fixes for `pkg/scheduler/apis/config/v1alpha1`

### PR DESCRIPTION
**Pull request by bagrat**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Minor linter cleanup.

**Which issue(s) this PR fixes**:
Part of https://github.com/kubernetes/kubernetes/issues/68026.

**Special notes for your reviewer**:
N/A

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/scheduler/apis/config/v1alpha1/defaults.go [hunk 1]
```diff
@@ -38,8 +38,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-// SetDefaults_KubeSchedulerConfiguration sets additional defaults
-func SetDefaults_KubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
+// SetDefaultsKubeSchedulerConfiguration sets additional defaults
+func SetDefaultsKubeSchedulerConfiguration(obj *kubescedulerconfigv1alpha1.KubeSchedulerConfiguration) {
```
Comment (SataQiu) [inline pkg/scheduler/apis/config/v1alpha1/defaults.go]: Thanks @bagrat 
But please do not modify these `SetDefaults_XXXX` functions.
This naming format has a special meaning and is used for automatic code generation.

Ref: https://github.com/kubernetes/code-generator/blob/master/cmd/conversion-gen/generators/conversion.go#L80

**Review by luxas — COMMENTED** [review]
Hi :wave:!

Thanks for this, but unfortunately we can't change this naming convention at the moment, due to how the machinery is made.

Can you please send some lint fixes for an other package instead, as we can't accept this particular patch?
If you need help, just contact me on Slack!

**Comment by SataQiu** [comment]
@bagrat  You can follow this to send some lint fixes #68026

**Pull request closed without merging**